src/ucs/time/time.h

```diff
@@ -36,6 +36,8 @@ typedef uint32_t             ucs_short_time_t;
 #define UCS_MSEC_PER_SEC   1000ull
 /* Micro per sec */
 #define UCS_USEC_PER_SEC   1000000ul
+/* Micro per milli */
+#define UCS_USEC_PER_MSEC 1000ull
```

Comment:
IMO no need to define it, can use the other constants

Reply:
fixed